src/modules/task/task.service.ts

```diff
@@ -1,22 +1,71 @@
 import { injectable } from 'inversify';
+import { TaskEntity } from '../../database/entities';
+import { NotFoundException } from '../../exceptions';
 import logger from '../../logger/pino.logger';
-import { CreateTaskDto, TaskIdDto } from './dto';
+import { CreateTaskDto, TaskIdDto, UpdateTaskDto } from './dto';
 
 @injectable()
 export class TaskService {
-  create(dto: CreateTaskDto) {
+  // Создание задачи
+  async create(dto: CreateTaskDto) {
     logger.info('Создание новой задачи');
-    return { id: 1, ...dto };
+
+    const task = await TaskEntity.create({
+      title: dto.title,
+      description: dto.description,
+    });
+    return task;
   }
 
-  getTasks() {
-    logger.info('Список задач new');
-    return { id: 1 };
+  // Получить задачу по ID
+  async getTaskId(data: TaskIdDto) {
+    logger.info(`Получение новой задачи  по id ${data.id}`);
+    const task = await TaskEntity.findOne({
+      where: { id: data.id },
+    });
+    if (!task) {
+      throw new NotFoundException(`Задачи с ID = ${data.id} не найдено`);
+    }
+    return task;
```

Comment:
Если аргумент упростить просто до id, кол-во кода чуть уменьшится
```suggestion
    logger.info(`Получение новой задачи  по id ${id}`);
    const task = await TaskEntity.findOne({
      where: { id },
    });
    if (!task) {
      throw new NotFoundException(`Задачи с ID = ${id} не найдено`);
    }
    return task;
```